src/libs/cluster-utils.js

```diff
@@ -83,6 +83,9 @@ export const trimAppsOldestFirst = _.flow(
   _.remove({ status: 'DELETING' }),
   _.sortBy('auditInfo.createdDate'))
 
+// TODO: Actually calculate app cost and pass in app
+export const appCost = () => appPrice / 730
```

Comment:
It looks like the apps we get back from Leo already have a `kubernetesRuntimeConfig`, so we could actually compute the real cost if we have an app. If we did that, then we'd just need to fabricate a runtime config with the known defaults when we want to show the estimated cost (before you actually create one)

Reply:
I was under the impression that we were punting on developing that for the time being to reduce scope. If we want to do the full calculation, that's totally fine, just want to make sure we are being intentional about effort

Reply:
That's fine too. I originally thought that we didn't have the data available, but when I was trying to reverse engineer where that number was coming from, I noticed we do get it.

In any case, if you buy my argument from the other comment about hourly rate, then this could be:
```js
_.find({ name: 'n1-standard-8' }, machineTypes).price
```
which is still hardcoded but documents where it's coming from.

Reply:
Oh, sure, that's easy.